packages/react-charts/src/components/ChartBullet/ChartBulletGroupTitle.tsx

```diff
@@ -137,10 +137,14 @@ export const ChartBulletGroupTitle: React.FunctionComponent<ChartBulletGroupTitl
   };
 
   const labelPadding = {
-    bottom: getPaddingForSide('bottom', padding, Number(theme.legend.style.labels.padding)),
-    left: getPaddingForSide('left', padding, Number(theme.legend.style.labels.padding)),
-    right: getPaddingForSide('right', padding, Number(theme.legend.style.labels.padding)),
-    top: getPaddingForSide('top', padding, Number(theme.legend.style.labels.padding))
+    bottom: getPaddingForSide(
```

Comment:
They change the props of `labels` from `VictoryStyleObject` to `VictoryLabelStyleObject | VictoryLabelStyleObject[]`. However, we cannot find a key named `padding` in an Array. So we need to cast it to `VictoryLabelStyleObject`. (Are there any better way to solve this problem?)

Reply:
I don't know if casting to `VictoryLabelStyleObject` would be correct as they are very different objects? I would be tempted to cast as `any` until `padding` is supported.

Reply:
Casting to VictoryLabelStyleObject is probably ok, but the alternative would be to cast as `any`.

Reply:
Casting to VictoryLabelStyleObject is probably ok, but the alternative would be to cast as any.